models/products/scratch/bigquery/snowplow_ecommerce_product_interactions_this_run.sql

```diff
@@ -7,7 +7,7 @@
 
 with product_info as (
   select
-    {{ dbt_utils.surrogate_key(['t.event_id', 'r.id']) }} as product_event_id,
+    {{ dbt_utils.generate_surrogate_key(['t.event_id', 'r.id']) }} as product_event_id,
```

Comment:
Yeah I also thought we should move away from the legacy behaviour and I figured that since we only just released support for Snowflake + Databricks better to rip the band-aid off earlier rather than later. Happy to include it though, WDYT?

Reply:
Agreed